src/Microsoft.OpenApi.OData.Reader/OpenApiConvertSettings.cs

```diff
@@ -105,28 +105,41 @@ public class OpenApiConvertSettings
         /// </summary>
         public bool EnableDiscriminatorValue { get; set; } = false;
 
+        /// <summary>
+        /// Gets/sets a value indicating whether or not to show the derived types of a reference in the responses payload.
+        /// </summary>
+        public bool ShowDerivedTypesReferencesForResponses { get; set; } = false;
```

Comment:
Maybe use the same pattern for the configuration name?  "Show...." => Enable....

Reply:
Effected